src/consensus/consensus.h

```diff
@@ -6,11 +6,105 @@
 #ifndef BITCOIN_CONSENSUS_CONSENSUS_H
 #define BITCOIN_CONSENSUS_CONSENSUS_H
 
+#include "amount.h"
+#include "consensus/params.h"
+
+#include <stdint.h>
+
+class CBlock;
+class CBlockHeader;
+class CBlockIndex;
+class CCoinsViewCache;
+class CTransaction;
+class CValidationState;
+
+/**
+ * Consensus validations:
+ * Check_ means checking everything possible with the data provided.
+ * Verify_ means all data provided was enough for this level and its "consensus-verified".
+ */
+namespace Consensus {
+
+/** Transaction validation functions */
+
+/**
+ * Context-independent CTransaction validity checks
+ */
+bool CheckTx(const CTransaction& tx, CValidationState& state, const Params& consensusParams);
+/**
```

Comment:
from IRC:

<dcousens> jtimon: the doc blocks padded line is a standard used throughout hte codebas
<dcousens> the codebase*
<dcousens> It makes it easier to see blocks
<dcousens> Rather than just 1 contiguous piece of code
<jtimon> dcousens: this "standard" is not documented in the style guide and it's not followed in many places (for example, main.h)
<dcousens> jtimon: I only opened 3-4 files before I said that,  maybe my sample size was too small
<dcousens> even in main.h, its pretty consistently there except for the 1 liners
<dcousens> I'd say its 50/50 in that file, with a higher prevalance in other files
<jtimon> we should document this if it's part of the style (unfortunately I don't think clang-format can enforce it)
<dcousens> jtimon: maybe just be consistent with what is already in the file
<dcousens> jtimon: i've lost the PR link, but, I'll leave that up to you
<dcousens> its a nit anyway
<jtimon> not opposing to it, but some people are already verifying the move-only commit and I would prefer not to modify the hash of the commit unless I need to for some other reason
<jtimon> well, there's not much in consensus/consensus.h before this PR...
<dcousens> :thumbs_up:, just mention that, the inconsistencies and that at some point we should unify it
<dcousens> its not relevant to your PR if its inconsistent elsewhere